src/main/kotlin/string/calculator/part/Splitter.kt

```diff
@@ -0,0 +1,22 @@
+package string.calculator.part
+
+import string.calculator.dto.SplitInput
+import string.calculator.domain.Operator
+import string.calculator.util.isEven
+import string.calculator.util.isOdd
+
+class Splitter {
+    fun split(input: String): SplitInput {
+        val splitInput = input.split(DELIMITER)
+        val numbers = splitInput.filterIndexed { index, _ -> index.isEven() }
+            .map { it.toLong() }.toMutableList()
+        val operators = splitInput.filterIndexed { index, _ -> index.isOdd() }
+            .map { Operator.of(it) }.toMutableList()
+
+        return SplitInput(numbers, operators)
```

Comment:
지금같이 단순히 두 개의 값을 한번에 저장할 자료형이 필요하다면 Pair 객체를 사용해볼 수도 있을 것 같아요. 😃
(이런 방식도있다 정도로 넘어가주시면 될 것 같습니다. 🤗)

```suggestion
        return Pair(numbers, operators)
```

infix 함수인 `to`를 활용해서 Pair 객체를 생성해볼 수도 있을 것 같아요. 😃
```suggestion
        return numbers to operators
```